### PR TITLE
Fix extending EthereumCallFilter

### PR DESCRIPTION
**Pull request by olibearo**
Addresses https://github.com/graphprotocol/support/issues/42

### CODE REVIEW

File: graph/src/components/ethereum/adapter.rs [hunk 1]
```diff
@@ -326,12 +326,21 @@ impl EthereumCallFilter {
...
+            match self.contract_addresses_function_signatures.get_mut(&key) {
+                Some((existing_start_block, existing_sigs)) => {
+                    *existing_start_block =
+                        cmp::min(proposed_start_block, existing_start_block.clone());
+                    existing_sigs.extend(new_sigs);
+                }
+                None => {
+                    self.contract_addresses_function_signatures
+                        .insert(key, (proposed_start_block, new_sigs));
+                }
```
Comment (Jannis) [inline graph/src/components/ethereum/adapter.rs]: We should call `key` `address` in this method. I got confused because I didn't know what `key` meant.

Comment (Jannis) [reply graph/src/components/ethereum/adapter.rs]: The code looks good otherwise. It would be nice to have a test for this though.

Comment (olibearo) [reply graph/src/components/ethereum/adapter.rs]: Addressed ;)

**Comment by olibearo** [comment]
Okay, let's add a test

**Comment by olibearo** [comment]
Test added @Jannis

**Pull request merged**